### PR TITLE
add allowIdleConsumers documentation

### PR DESCRIPTION
**Pull request by lionelvillard**
Documentation for this proposed new feature: https://github.com/kedacore/keda/pull/1684

Relates to https://github.com/kedacore/keda/issues/1683

### CODE REVIEW

**Review by tomkerkhove — APPROVED** [review]

**Comment by tomkerkhove** [comment]
Waiting for feature PR to be merged but LGTM, thanks!

File: content/docs/2.3/scalers/apache-kafka.md
Comment (zroubalik) [inline content/docs/2.3/scalers/apache-kafka.md:40]:
```suggestion
- `allowIdleConsumers` - Optional. When set to `true`, the number of replicas can exceed the number of
partitions on a topic, allowing for idle consumers. (default: `false`)
```

**Review by zroubalik — CHANGES_REQUESTED** [review]

**Review by zroubalik — APPROVED** [review]
LGTM

**Pull request merged**
